### PR TITLE
DDP-5850 translate MBC more details page to Spanish

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 1]
```diff
@@ -339,112 +339,113 @@
...
                 },
-                "Section2": "[es]Our goal is to reach out to patients with metastatic breast cancer throughout the USA and Canada in order to study their cancer using cutting-edge genomic and molecular research studies performed at the Broad Institute by allowing patients to participate in these studies regardless of where they live. We are also asking patients from all over the world to sign up so that we can collect information worldwide and eventually expand to additional countries."
+                "Section2": "Nuestra meta es darles a pacientes alrededor de los Estados Unidos y sus territorios y Canadá con cáncer de mama metastásico la oportunidad de participar en estos estudios sin importar donde vivan para poder analizar su cáncer usando estudios moleculares y genómicos de última generación llevados a cabo en el Instituto Broad (Broad Institute)."
             },
             "HowProjectWorks": {
-                "Title": "[es]Here’s how the project works:",
+                "Title": "Cómo Funciona el Proyecto:",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json]: Do we need keeping every word capitalized  there?

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 2]
```diff
@@ -339,112 +339,113 @@
...
-                    "Pt1": "[es]The Metastatic Breast Cancer Project takes a new approach to cancer research in which researchers partner directly with patients, who share their samples and clinical information in order to speed important discoveries. The MBCproject is part of",
-                    "Pt2": "[es]a nonprofit organization that brings together patients and researchers as partners to accelerate discoveries in cancer research.",
-                    "Pt3": "[es]is stewarded by four leading organizations: Emerson Collective, a California-based social change organization; the Broad Institute of MIT and Harvard, a leading nonprofit biomedical research institution; the Biden Cancer Initiative, an independent nonprofit organization that builds on the federal government’s Cancer Moonshot; and the Dana-Farber Cancer Institute, a leading cancer hospital."
+                    "Pt1": "El Metastatic Breast Cancer Project es un nuevo modelo en la investigación del cáncer en el cual los investigadores colaboran directamente con pacientes, quienes comparten sus muestras de tumor e información clínica para juntos acelerar descubrimientos. El MBCproject es parte de",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:345]: I'd suggest replacing `Project` with `proyecto`
(at the beginning and at the end of the phrase).

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 3]
```diff
@@ -339,112 +339,113 @@
...
+                    "Question": "8. ¿Esta investigación se realiza con fines de lucro?",
+                    "Text": "No. Este proyecto es un esfuerzo sin fines de lucro. Nuestra misión es generar, compilar y compartir libremente datos con la comunidad de investigación de forma que se proteja su información personal para acelerar descubrimientos científicos en cáncer."
                 },
                 "InfoForPhysicians": {
-                    "Question": "[es]9. Is there information that I can bring to my doctor so that I can discuss this study with them?",
-                    "Text": "[es]Yes, you can print out or use your mobile device to share the document linked here:",
-                    "Link": "[es]Metastatic Breast Cancer Project For Physicians"
+                    "Question": "9. ¿Hay alguna información que pueda proporcionar a mi médico y/u hospital para poder hablar sobre este estudio con ellos?",
+                    "Text": "Sí, puede imprimir o usar su dispositivo móvil para compartir este documento:",
+                    "Link": "The Metastatic Breast Cancer Project para Médicos"
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json]: `El Metastatic Breast Cancer proyecto para Médicos`
seems to be more proper

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 4]
```diff
@@ -339,112 +339,113 @@
             "SubmitButton": "[es]OK"
         },
         "MoreDetails": {
-            "Title": "[es]Learn more about the Metastatic Breast Cancer Project",
+            "Title": "Aprende más sobre el Metastatic Breast Cancer Project",
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:342]: `Project` => `Proyecto` ?

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json]: hmm, good catch! copied from the file, but I guess we don't need to capitalize it

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json (hunk 2 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:345]: I'm not sure about that... I use the provided translation, probably it was important to keep the original name of the project? as you can notice other words in the name are not translated as well

Comment (alextanlan) [reply ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:345]: Ok, let's keep it as is. 
Perhaps, translators will take a look at the completed work and polish smth.

**Review by alextanlan — APPROVED** [review]

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json (hunk 3 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json]: I replaced The with El

**Pull request merged**